examples/niscope/graph-measurement.py

```diff
@@ -42,11 +42,21 @@
 
 channels = "0"
 
-def CheckStatus(scope_service, vi, result):
-    if (result.status != 0):
-        error_result = scope_service.GetErrorMessage(niscope_types.GetErrorMessageRequest(vi = vi, error_code = result.status))
-        print(error_result.error_message)
-        exit()
+# Checks for errors. If any, throws an exception to stop the execution.
+any_error = False
+def CheckForError (vi, status) :
+    global any_error
+    if(status != 0 and not any_error):
+        any_error = True
+        ThrowOnError (vi, status)
+
+def ThrowOnError (vi, error_code):
+    error_message_request = niscope_types.ErrorMessageRequest(
```

Comment:
Earlier we were using `GetErrorMessage` for retrieving error. Is change from that to using `scope_service.ErrorMessage` API intentional?

Reply:
While copy-pasting from this portion from DCPower, I did not realize Scope was using GetErrorMessage instead of simply ErrorMessage.
Done changing at all places.